src/puppetlabs/trapperkeeper/core.clj

```diff
@@ -32,13 +32,13 @@
   (`init`, `start`, `stop`) as you see fit;  if you'd like to have the trapperkeeper
   framework block the main thread to wait for a shutdown event, call `init`,
   `start`, and then `run-app`."
-  [services config-data]
+  [services config-data-fn]
```

Comment:
That's fair; I meant to revisit this and possibly do some type checking on this argument.  We will either amend this PR or file a follow-up ticket that is addressed before a release, thanks for pointing it out.